skprocrustes/tests/test_skprocrustes.py

```diff
@@ -4,17 +4,16 @@
                            assert_almost_equal, assert_warns, assert_array_less)
 from scipy import linalg as sp
 import skprocrustes as skp
-import nose
```

Comment:
`nose` needs to be present but not is imported. The rest of the changes in this file are blank space noise. Sorry :grimacing: